source/_components/cast.markdown

```diff
@@ -30,9 +30,58 @@ For example, Cast devices can only be discovered if they are on the same subnet
 cast:
   media_player:
   - host: 192.168.1.10
+  - port: 8009
 ```
 
 Configuration variables:
 
 - **host** (*Optional*): Use only if you don't want to scan for devices.
```

Comment:
Host is required according to the code

Reply:
Host is required *if* manually specifying chromecasts.

Just spotted an error however; `port` may not have a `-`.